content/influxdb/cloud-iox/migrate-data/migrate-tsm-to-iox.md

```diff
@@ -0,0 +1,389 @@
+---
+title: Migrate data from TSM to IOx in InfluxDB Cloud
+description: >
+  To migrate data from a TSM-backed InfluxDB Cloud organization to an InfluxDB
+  IOx-backed organization, query the data in time-based batches and write the
+  queried data to an IOx bucket in your InfluxDB Cloud organization.
+menu:
+  influxdb_cloud_iox:
+    name: Migrate from TSM to IOx
+    parent: Migrate data
+weight: 102
+---
+
+To migrate data from an InfluxDB Cloud organization backed by TSM to an organization
+backed by InfluxDB IOx, query the data from your TSM backed buckets in time-based
+batches and write the queried data to a bucket in your InfluxDB IOx-backed organization.
+Because full data migrations will likely exceed your organizations' limits and
+adjustable quotas, migrate your data in batches.
+
+The following guide provides instructions for setting up an InfluxDB task
+that queries data from an InfluxDB Cloud TSM-backed bucket in time-based batches
+and writes each batch to another InfluxDB Cloud IOx-backed bucket in another
+organization.
+
+{{% cloud %}}
+All query and write requests are subject to your InfluxDB Cloud organization's
+[rate limits and adjustable quotas](/influxdb/cloud-iox/account-management/limits/).
+{{% /cloud %}}
+
+- [Set up the migration](#set-up-the-migration)
+- [Migration task](#migration-task)
+  - [Configure the migration](#configure-the-migration)
+  - [Migration Flux script](#migration-flux-script)
+  - [Configuration help](#configuration-help)
+- [Monitor the migration progress](#monitor-the-migration-progress)
+- [Troubleshoot migration task failures](#troubleshoot-migration-task-failures)
+
+## Set up the migration
+
+{{% note %}}
+The migration process requires two buckets in your source InfluxDB
+organization—one bucket to store the migrated data and another bucket to store migration metadata.
+If the destination organization uses the [InfluxDB Cloud Free Plan](/influxdb/cloud/account-management/limits/#free-plan),
+any buckets in addition to these two will exceed the your plan's bucket limit.
```

Comment:
Explicitly recommend upgrading or state that it's not possible within the free plan?